client/src/components/Navbar/Navbar.js

```diff
@@ -15,13 +15,13 @@ class Navbar extends Component {
     this.state = {
```

Comment:
Do we need these changes? They seem unrelated to what you're working on? Can you maybe reset this file so these changes aren't part of this PR?